src/WhiteLibrary/keywords/items/listview.py

```diff
@@ -339,6 +376,10 @@ def _get_row_by_index(self, locator, index):
         listview = self.state._get_typed_item_by_locator(ListView, locator)
         return listview.Rows.Get(int(index))
 
+    def _get_row_by_text(self, locator, text):
+        listview = self.state._get_typed_item_by_locator(ListView, locator)
+        return next((row for row in listview.Rows if row.Cells[0].Text == text), None)
```

Comment:
The search currently returns None if no matching row is found (like the other methods to get rows/cells do). Error messages for all cases where an item is needed and not found should be more descriptive.

Reply:
if there are already keywords with similar issue, it doesnt really matter much if new ones are added.